platforms/pictique/client/src/lib/fragments/BottomNav/BottomNav.svelte

```
@@ -53,14 +72,26 @@
-			}
+			(async () => {
+				try {
+					uploadedImages.value = await Promise.all(
+						selectedFiles.map(async (file) => ({
+							url: await readFileAsDataUrl(file),
+							alt: file.name,
+							size: file.size
+						}))
+					);
+					if (uploadedImages.value.length > 0) {
+						goto('/post');
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)